pkg/scheduler/framework/plugins/volumezone/volume_zone_test.go

```diff
@@ -493,11 +510,229 @@ func TestWithBinding(t *testing.T) {
 			state := framework.NewCycleState()
 			node := &framework.NodeInfo{}
 			node.SetNode(test.Node)
+			translator := csitrans.New()
+			p := &VolumeZone{
+				pvLister:  pvLister,
+				pvcLister: pvcLister,
+				scLister:  scLister,
+
+				csiNodeLister:            nil,
+				translator:               translator,
+				csiMigratedPluginManager: csimigration.NewPluginManager(translator, utilfeature.DefaultFeatureGate),
+			}
+			_, preFilterStatus := p.PreFilter(ctx, state, test.Pod)
+			if diff := cmp.Diff(preFilterStatus, test.wantPreFilterStatus); diff != "" {
+				t.Errorf("PreFilter: status does not match (-want,+got):\n%s", diff)
+			}
+			filterStatus := p.Filter(ctx, state, test.Pod, node)
+			if diff := cmp.Diff(filterStatus, test.wantFilterStatus); diff != "" {
```

Comment:
```suggestion
			if diff := cmp.Diff(test.wantFilterStatus, filterStatus); diff != "" {
```

to match `(-want,+got)`

Same in the diff above